packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -177,6 +224,10 @@ const RelationModalWrapper = ({ relation, triggerButtonLabel }: RelationModalPro
                         if (modified && !isSubmitting) {
                           setIsConfirmationOpen(true);
                         } else {
+                          // Add current relation to history before opening a new one
+                          if (currentDocumentMeta && Object.keys(currentDocumentMeta).length > 0) {
```

Comment:
I'm not sure we need to check if the object is empty or not, in fact currentDocumentMeta should never be undefined so this check may be unnecessary

```suggestion
                          if (currentDocumentMeta) {
```

Reply:
Yes, I agree with you currentDocumentMeta has always a value